src/loaders/genome_collection/parse_computed_genome_attribs.py

```diff
@@ -26,23 +26,30 @@
                         Extract results from tools. (default: retrieve all available sub-directories in the [load_ver]
                         directory)
   --root_dir ROOT_DIR   Root directory for the collections project. (default: /global/cfs/cdirs/kbase/collections)
-  -o OUTPUT, --output OUTPUT
-                        output JSON file path
 
 """
 import argparse
 import copy
 import os
 import sys
+from typing import Dict, Union, List, Any
 
 import pandas as pd
 
+import src.common.storage.collection_and_field_names as names
+from src.common.storage.db_doc_conversions import collection_data_id_key, collection_load_version_key
 from src.loaders.common import loader_common_names
 from src.loaders.common.loader_helper import convert_to_json, init_genome_atrri_doc, merge_docs
+from src.service.data_products.heatmap_common_models import HeatMapMeta, ColumnInformation, ColumnCategory, Cell, \
+    HeatMapRow, ColumnType
 
 # Default result file name for parsed computed genome attributes data for arango import.
-# Collection and load version information will be prepended to this file name.
-COMPUTED_GENOME_ATTR_FILE = "computed_genome_attribs.json"
+# Collection, load version and tools name will be prepended to this file name.
+COMPUTED_GENOME_ATTR_FILE = "computed_genome_attribs.jsonl"
+
+# Default result file name for parsed heatmap data for arango import.
+# Collection, load version and tools name will be prepended to this file name.
```

Comment:
Commented here before seeing that this is really a filename stuffix and the datatype is prepended. I think the if the comment notes it's a suffix rather than a name that'll do

Reply:
👍

Reply:
:+1: